r/src/R/ai/h2o/sparkling/InternalBackendConf.R

```diff
@@ -33,8 +33,6 @@ InternalBackendConf <- setRefClass("InternalBackendConf", methods = list(
 
     subseqTries = function() { invoke(jconf, "subseqTries") },
 
-    h2oNodeWebEnabled = function() { invoke(jconf, "h2oNodeWebEnabled") },
```

Comment:
We don't need to deprecate as this was introduced in the same version as where we are deprecating  this option. Was never exposed to users